### ISSUE TITLE
Question on PULoss function

### ISSUE THREAD

**Issue by ccccly** [lejeunel/ksptrack#6]
Hi lejeunel!

I recently ran the code you provided and found that at line 100 in the losses.py file, the code says loss = pos_risk-neg_risk, but what I understand from the ssnnpu article should be loss = pos_risk+neg_risk, am I misunderstanding?

Thank you in advance !
